src/pages/Admin/AdminEditComponent.tsx

```diff
@@ -515,12 +515,12 @@ const PlacementForm: React.FC<PlacementFormProps> = (props) => {
             {formError && (
               <div className="input-error">
                 <div>One or more errors occurred when saving the form:</div>
-                {Object.keys(errors).map((fieldName) => (
-                  <div>
+                {Object.keys(errors).map((fieldName, index) => (
+                  <div key={index}>
```

Comment:
[NB, nit] I think this would be fixed better by using `fieldName`? (assuming we only report a single error per field?)